apps/docs/src/@primer/gatsby-theme-doctocat/components/header.module.css

```
@@ -0,0 +1,46 @@
+.PageHeader {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: var(--base-size-16) var(--base-size-24) var(--base-size-16)
+    var(--base-size-24);
+  border-bottom: 1px solid var(--borderColor-default, #d1d9e0);
+  background: var(--brand-color-canvas-default);
+  z-index: 1;
+}
+
+.SiteTitle {
+  display: flex;
+  gap: var(--base-size-8);
+  align-items: center;
+  text-decoration: none;
+}
+
+.SiteTitle svg {
+  fill: var(--brand-color-text-default);
+}
+
+.Title {
+  display: inline-block;
+  font-size: var(--text-body-size-large);
```

Size is really small. Switching to a Primer Brand token should fix this ☝️ 

```suggestion
  font-size: var(--brand-text-size-400);
```